PatcherLoader/Loader.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.IO;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using System.Reflection;
+using Mono.Cecil;
+using BepInEx;
+using BepInEx.Logging;
+
+namespace PatcherLoader
+{
+    /// <summary>
+    /// This "patcher" doesn't patch any .NET assemblies. It just loads and
+    /// executes a native DLL, which applies some patches to in-memory native
+    /// code.
+    ///
+    /// This needs to be implemented as a preloading-time patcher rather than
+    /// a regular plugin because the latter doesn't run early enough in the
+    /// startup sequence of the game.
+    /// </summary>
+    public static class Loader
+    {
+        public static IEnumerable<string> TargetDLLs => GetDLLs();
+
+        public static void Patch(AssemblyDefinition assembly)
+        {
+        }
+
+        private readonly static ManualLogSource logger
+            = Logger.CreateLogSource("KK_MainGameVR_Patcher");
+
+        private delegate void SetupAll();
+
+        private static IEnumerable<string> GetDLLs()
+        {
+            var processName = Paths.ProcessName;
+            if (processName != "Koikatu" && processName != "Koikatsu Party")
+            {
+                yield break;
+            }
```

Comment:
Can you make the loader do nothing if VR mode is not going to be enabled? The idea is to have as little effect on the game as possible in that case, so that any potential bugs do not affect users outside of VR.

Reply:
It only boots config and cross fader outside of vr, same as kks. Should be safe and even preferable, as changing config in vr is a bit of a pain, your call though.

Reply:
probably, not exactly sure of it to be honest

Reply:
ok, i've no clue what it does at all

Reply:
done